routing/src/cpi.rs

```diff
@@ -325,7 +325,8 @@ fn handle_request(
 
     // ignore additions if have no config. Connects are allowed, so are deletions to wipe out old state
     if !db.have_config() && op == RpcOp::Add {
-        debug!("Ignoring request: no config is available");
+        error!("Ignoring request: there's no config. This should not happen...");
+        error!("... but won't cause malfunction. Trust the 🦔's");
```

Comment:
Would you please consider removing the last part of the comment?
- If it _does_ cause malfunction because of whatever bug, it's not good for trust, and might create frustration with users seeing this.
- I would also recommend against tying (even more) the open-source project to the vendor company. Others could work on this in the future and not be part of Hedgehog.

```suggestion
        error!("Ignoring request: there's no config. This should not happen, but won't cause malfunction if it does.");
```

Reply:
:cry: I like hedgehogs too! But I believe it's for the best. Thanks!